main.py

```diff
@@ -452,6 +507,18 @@ def on_mqtt(self, client, userdata, msg):
             # data is a dict with the following keys: rtu_id, board, route, lat, lon, speed, dir
             self.refresh_transport(data)
 
+    @staticmethod
+    @run_async
+    def send_message_hook(chat_id, text):
+        """This will be invoked by the REST API when the sysadmin wants to
+        send a message back to a user who left feedback via /feedback and
+        asked a question, which expects a response
+        :param chat_id: int, chat identifier
+        :param text: str, the text to be sent to the user"""
+        global bot
```

Comment:
Don't really like this 😬 can we do something about it?

Reply:
You're right. Done.